packages/server/src/fhir/routes.ts

```diff
@@ -23,9 +23,12 @@ import { sendOutcome } from './outcomes';
 import { Repository } from './repo';
 import { rewriteAttachments, RewriteMode } from './rewrite';
 import { smartConfigurationHandler, smartStylingHandler } from './smart';
+import { getConfig } from '../config';
 
 export const fhirRouter = Router();
 
+let router: FhirRouter;
```

Comment:
If we're going to make this a global for the file, we should give it a more descriptive name (i.e., `internalFhirRouter` or something like that).  Especially when it's sitting right next to `fhirRouter`.